calibrators/marker_radar_lidar_calibrator/README.md

```diff
@@ -0,0 +1,141 @@
+| `input_radar_msg`        | `radar_msgs::msg::RadarTracks`  | Radar message for calibration, `input_radar_msg` is defined in the launcher.           |
+
+### Output
+
+| Name                          | Type                                   | Description                                               |
+| ----------------------------- | -------------------------------------- | --------------------------------------------------------- |
+| `lidar_background_pointcloud` | `sensor_msgs::msg::PointCloud2`        | Publishes the background pointcloud from lidar.           |
+| `lidar_foreground_pointcloud` | `sensor_msgs::msg::PointCloud2`        | Publishes the foreground pointcloud from lidar.           |
+| `lidar_colored_clusters`      | `sensor_msgs::msg::PointCloud2`        | Publishes colored pointcloud clusters from lidar.         |
+| `lidar_detection_markers`     | `visualization_msgs::msg::MarkerArray` | Publishes lidar detections.                               |
+| `radar_background_pointcloud` | `sensor_msgs::msg::PointCloud2`        | Publishes the background pointcloud from radar.           |
+| `radar_foreground_pointcloud` | `sensor_msgs::msg::PointCloud2`        | Publishes the foreground pointcloud from radar.           |
+| `radar_detection_markers`     | `visualization_msgs::msg::MarkerArray` | Publishes radar detections.                               |
+| `matches_markers`             | `visualization_msgs::msg::MarkerArray` | Publishes matched lidar and radar detection.              |
+| `tracking_markers`            | `visualization_msgs::msg::MarkerArray` | Publishes tracks of reflectors.                           |
+| `text_markers`                | `visualization_msgs::msg::Marker`      | Publishes text markers that show the calibration metrics. |
+| `calibration_metrics`         | `std_msgs::msg::Float32MultiArray`     | Publishes calibration metrics.                            |
+
+### Services
+
+| Name                       | Type                                                  | Description                                                                              |
+| -------------------------- | ----------------------------------------------------- | ---------------------------------------------------------------------------------------- |
+| `extrinsic_calibration`    | `tier4_calibration_msgs::` `srv::ExtrinsicCalibrator` | Generic calibration service. The call is blocked until the calibration process finishes. |
+| `extract_background_model` | `std_srvs::srv::Empty`                                | Strat to extract the background model from radar and lidar data.                         |
+| `add_lidar_radar_pair`     | `std_srvs::srv::Empty`                                | User is able to click this button to add lidar-radar pair.                               |
```

Comment:
The use of the word "User" is not correct. Nor the description matches the style of the other services

Reply:
fixed in https://github.com/tier4/CalibrationTools/pull/164/commits/2f19766f0942fb7fd3dadfdf347258448d7ae243

Reply:
`Add lidar-radar pair for calibration` is not correct.
Either 
add a lidar-radar pair...
or
add lidar-radar pairs...

In this case both are correct and incorrect at the same time, since the algorithm tries to add several but in practice only adds a pair (we have never tried with multiple corner reflectors)

Reply:
fixed in https://github.com/tier4/CalibrationTools/pull/164/commits/55f9b0d45b29d709aab2193b6f785e7fd738f806